ddtrace/profiling/exporter/pprof.pyx

```diff
@@ -406,22 +431,32 @@ class PprofExporter(exporter.Exporter):
     def _lock_event_group_key(
         self, event  # type: lock.LockEventBase
     ):
-        if event.trace_type == ext.SpanTypes.WEB.value:
-            trace_resource = self._none_to_str(event.trace_resource)
-        else:
-            # Do not export trace_resource for privacy concerns.
-            trace_resource = ""
+        trace_id = None
```

Comment:
The coding here is almost identical to `_stack_event_group_key`. Perhaps this could be put into a helper method that returns all but the first two entries of the returned tuple and here do
~~~ python
    return (event.lock_name, event.thread_id) + self._common_group_key(event)
~~~

Reply:
I'm not really familiar with this code. Would it be ok if we postpone this refactoring?

Reply:
Sure, we can sort this out later!